modules/openapi-generator/src/main/resources/JavaSpring/api.mustache

```diff
@@ -150,7 +150,7 @@ public interface {{classname}} {
         {{/vendorExtensions.x-tags.size}}
         responses = {
             {{#responses}}
-            @ApiResponse(responseCode = "{{{code}}}", description = "{{{message}}}"{{#baseType}}, content = {
+            @ApiResponse(responseCode = {{#isDefault}}"default"{{/isDefault}}{{^isDefault}}"{{{code}}}"{{/isDefault}}, description = "{{{message}}}"{{#baseType}}, content = {
```

Comment:
Code change looks great! Should there be a unit test for this? I'm also happy to add it if you think it adds value

Reply:
well, samples are updated. do you think we still need unit test for this? If yes I can add them quickly

Reply:
Sorry missed this. I do think unit tests still add value because if someone breaks it, the unit tests will block the submission (hopefully) or alert them that the samples were updated incorrectly. I can add these in a few days and send out for review